packages/@react-spectrum/label/src/Label.tsx

```diff
@@ -29,6 +29,7 @@ function Label(props: SpectrumLabelProps, ref: DOMRef<HTMLLabelElement>) {
     labelAlign = labelPosition === 'side' ? 'start' : null,
     isRequired,
     necessityIndicator = isRequired != null ? 'icon' : null,
+    includeNecessityIndicatorInAccessibilityName = false,
```

Comment:
😱

Reply:
Do you have a better property name for me?